### ISSUE TITLE
Can not set 'close' event listener twice

### ISSUE THREAD

**Issue by siric** [ClusterWS/cWS#33]
I'm trying out the just released 1.0 version and I'm getting a `Error: Can not set 'close' event listener twice`. Connection and error listeners are working but it's hanging at the `close` event. I've seen you've made some adjustments to the event system, is there something I'm missing?

```js
const wss =  new WebSocket.Server(options);
wss.on('connection', (client, req) => this.onConnect(client, req));
wss.on('close', () => this.onClose());
wss.on('error', e => this.onError(e));
```

`uncaughtException: Error: Can not set 'close' event listener twice at WebSocketServer.on (@clusterws/cws/dist/server.js:81:19)`

**Comment by goriunov** [comment]
Oh yes, on `close` listener only applicable to the `client` (it was not there in previous versions ether) just now it throws error when you try to set something what does not exists (will need to add another listener validation step). 

You can handle on close at the moment as `cb`:
```js
wss.close(() => {
/// your logic
})
```

Will look in to adding this. Next in plan is to make `cws` more like `ws` module.

**Comment by goriunov** [comment]
Just released version 1.1.0 with support for on `close` event on the server. Also from now on it will print to `console` if  you are trying to set wrong (unsupported) event instead of not starting up

**Comment by siric** [comment]
Great, working as expected now!
